src/backend/src/utils/homepage-notifications.utils.ts

```diff
@@ -0,0 +1,37 @@
+import { getNotificationQueryArgs } from '../prisma-query-args/notifications.query-args';
+import prisma from '../prisma/prisma';
+import notificationTransformer from '../transformers/notification.transformer';
+import { NotFoundException } from './errors.utils';
+
+const sendNotificationToUser = async (userId: string, notificationId: string, organizationId: string) => {
+  const requestedUser = await prisma.user.findUnique({
+    where: { userId }
+  });
+
+  if (!requestedUser) throw new NotFoundException('User', userId);
+
```

Comment:
why are we checking that user exists but not the other things, not sure if we need to check any of these and mark in the documentation that verification of these items should exist already. Or we do all of the verification in here.